__tests__/app.test.js

```diff
@@ -89,6 +88,42 @@ describe('GET /api/articles/:article_id', () => {
         });
     })
 })
-});
-
-
+describe('GET /api.articles/:article_id/comments', () => {
+    test('status 200: responds with an array of comments for given article_id', () => {
+        return request(app)
+        .get('/api/articles/3/comments')
+        .expect(200)
+        .then((response) => {
+            const comments = response.body.comments;
+            console.log(response.body)
+            comments.forEach((comment) => {
```

Comment:
It would be great to check the length of the array is expected for this one to prevent any false positives before your loop